regression/pages/common/utils.py

```diff
@@ -0,0 +1,526 @@
+            """
+            Wrapper
+            Args:
+                self:
+                *args:
+                **kwargs:
+            """
+            test_function(self, *args, **kwargs)
+            self.full_cleanup = False
+        return wrapper
+    return decorator
+
+
+def fill_input_fields(page, css_selectors, values):
+    """
+    A helper function to fill multiple fields in a form
+    Args:
+        page:
+        css_selectors:
+        values:
+    """
+    for css_selector, value in izip(css_selectors, values):
+        page.q(css=css_selector).fill(value)
+
+
+def select_value_from_drop_down(page, drop_down_name, value):
+    """
+    A helper function to select value from drop down
+    Args:
+        page:
+        drop_down_name:
+        value:
+    """
+    page.wait_for_element_visibility(
+        'select[name=' + drop_down_name + ']', 'Drop down is visible')
+    page.q(
+        css='select[name=' + drop_down_name + '] option[value="{}"]'.format(
+            value)).click()
+    page.wait_for(
+        lambda: page.q(
+            css='select[name=' + drop_down_name + '] option[value="{}"]'.
+            format(value)).selected,
+        "Correct value is selected",
+        timeout=DEFAULT_TIMEOUT
+    )
+
+
+def select_values_from_drop_downs(page, drop_down_names, values):
+    """
+    A helper function to select values from multiple drop downs on a page
+    Args:
+        page:
+        drop_down_names:
+        values:
+    """
+    for drop_down_name, value in izip(
+            drop_down_names, values
+    ):  # pylint: disable=cell-var-from-loop
+        page.wait_for_element_visibility(
+            'select[name=' + drop_down_name + ']', 'Drop down is visible')
+        page.q(
+            css='select[name=' + drop_down_name + '] option[value="{}"]'.
+            format(value)).click()
+        page.wait_for(
+            lambda: page.q(
+                css='select[name=' + drop_down_name + '] option[value="{}"]'.
+                format(value)).selected,
+            "Correct value is selected",
+            timeout=DEFAULT_TIMEOUT
+        )
+
+
+def click_checkbox(page, css_selector):
+    """
+    A helper function to click check boxes
+    Args:
+       page:
+       css_selector:
+    """
+    # pylint: disable=unnecessary-lambda
+    page.wait_for_element_visibility(css_selector, 'wait for target checkbox')
+    page_elem = page.q(css=css_selector).results[0]
+    if not page_elem.is_selected():
+        page.browser.execute_script("arguments[0].click();", page_elem)
+        page.wait_for(
+            lambda: page_elem.is_selected(),
+            'Target checkbox is selected'
+        )
+
+
+def remove_spaces_from_list_elements(list_with_spaces):
+    """
+    A helper function to remove spaces from all items in the list
+    Args:
+      list_with_spaces:
+    """
+    return [x.replace(' ', '') for x in list_with_spaces]
+
+
+def get_text(page, css_selectors):
```

Comment:
this name is misleading. please rename it so it's clear that it is a bulk function that returns a list.